indra/assemblers/pybel_assembler.py

```diff
@@ -104,7 +104,26 @@ def make_model(self):
             else:
                 logger.info('Unhandled statement: %s' % stmt)
         return self.model
-
+    
+    def save_model(self, path, output_format=None):
+        """Saves the :class:`pybel.BELGraph` using one of the outputs from 
+        :py:mod:`pybel`
+
+        :param str path: The path to output to
+        :param Optional[str] output_format: Output format as ``cx``, 
+                            ``pickle``, ``json`` or defaults to ``bel``
+        """
```

Comment:
Can switch these docstrings to the numpy doc format used elsewhere in INDRA, for consistency?